lib/Map/Legend.js

```diff
@@ -115,36 +115,24 @@ var Legend = function(props) {
      */
     this.variableNameTop = defaultValue(props.variableNameTop, 17);
 
+    /**
+     * Gets or sets the CSS class that will be applied to the legend when it is displayed.
+     * This is used to ensure that the correct font is used when measuring text for word wrapping.
+     * @type {String}
+     */
+    this.cssClass = defaultValue(props.cssClass, 'legend__legend');
```

Comment:
Alex used `foreignObject` initially, but it doesn't work in Microsoft browsers, even new ones.  Also, it's hard to make the rest of the content move down to make room, with the result that you see in the bug that this one fixes.

Arguably the right solution is to let legends generate HTML instead of SVG.  I think that might  need to wait until after 15 July though.